src/doing/list/commands.py

```diff
@@ -78,7 +78,15 @@
     help="Output format. 'table' has a rich display, 'array' will return a string list with ID's.",
     show_envvar=True,
 )
-def list(assignee, author, label, state, type, web, story_points, output_format):
+@click.option(
+    "--show_state/--no-show_state",
+    required=False,
+    default=False,
+    type=bool,
+    help="Show column with work item state.",
```

Comment:
@ClaudioSalvatoreArcidiacono Thank you for the comments! 
`click` generates the documentation automatically. When writting `doing list --help`, you will see this `help` message. It will also show you the envvar that you can use to control the default value of this option. 
The help will look like this:

```bash
  --show_state / --no-show_state  Show column with work item state.  [env var:
                                  DOING_LIST_SHOW_STATE]
```

The documentation website will be automatically updated with this as well, upon release on github.
https://ing-bank.github.io/doing-cli/reference/manual/list/

What do you think?